apps/codeimage/src/state/presets/presets.ts

```diff
@@ -63,6 +66,9 @@ const PresetStoreDefinition = experimental__defineResource(fetchInitialState)
             : new Date(b.updatedAt).getTime() - new Date(a.updatedAt).getTime(),
         );
       },
+      canAddNewPreset() {
+        return isLogged() && this.sortedPresets().length <= PRESET_LIMIT;
```

Comment:
i think we can remove isLogged otherwise the button will be always disabled for guest users 🤔